### PR TITLE
Make sure pdbs are copied along with exe and dlls when bootstrapping

### PR DESCRIPTION
**Pull request by rylev**
This makes it easier to find the pdbs when wanting to debug the compiler on Windows.

### CODE REVIEW

File: src/bootstrap/util.rs [hunk 1]
```diff
@@ -32,6 +32,11 @@ pub fn is_dylib(name: &str) -> bool {
     name.ends_with(".dylib") || name.ends_with(".so") || name.ends_with(".dll")
 }
 
+/// Returns `true` if the file name given looks like a debug info file
+pub fn is_debug_info(name: &str) -> bool {
+    name.ends_with(".pdb")
```
Comment (Mark-Simulacrum) [inline src/bootstrap/util.rs:37]: I'm now wondering if this is also necessary for similar cases with split debuginfo on linux/macOS; could you add a comment here that "we may need to consider linux/macOS split debuginfo here too"?

**Review by Mark-Simulacrum — COMMENTED** [review]
r=me with comment added

**Comment by Mark-Simulacrum** [comment]
r=me with commits squashed

**Comment by ehuss** [comment]
Just a question of curiosity:  I was under the impression that executables contain an absolute path to the pdb file, and that the msvc tools will pick it up even if you move the exe but not the pdb (they will only search the current directory if the original was not found).  Is that not working in that way?

**Comment by rylev** [comment]
@ehuss That's correct as far as I understand that. I ran into this issue when the original pdbs had been deleted. This will just make housekeeping easier in some cases.

**Comment by Mark-Simulacrum** [comment]
I'd guess we also need to do this -- or a similar patch -- if we distributed rustc debuginfo on these targets.

@bors r+

**Comment by rylev** [comment]
This had the unintended but very much welcome side effect of including pdbs in the Rust distribution. PDBs are now distributed through rustup meaning that when debugging Rust binaries on Windows that link the standard library, std lib symbols are available.

**Pull request merged**